recipes-sota/ostree-initrd/files/init.sh

```diff
@@ -64,6 +64,10 @@ mount "$ostree_sysroot" /sysroot || {
 
 ostree-prepare-root /sysroot
 
+mkdir -p /sysroot/run
+#Move /run(initramfs) to /sysroot/run(real-rootfs) 
```

Comment:
nit, please add extra space between '#' and 'Move'

Also please add a comment explaining why this is needed (e.g. to avoid using a clean /run that is later mounted by systemd).